src/main/java/com/example/GymProject/GymProjectApplication.java

```diff
@@ -1,13 +1,19 @@
 package com.example.GymProject;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@SpringBootApplication
+import com.example.GymProject.config.AppConfig;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
This line is redundant. You have AppConfig annotated with the same - it will work.